docs/Runtime Parameters.md

```diff
@@ -50,7 +70,8 @@ accepted inputs: { true, false }
 #### scanningFrequency_Hz 
 The speed at which the sensor should spin at
 
-accepted inputs: { 7, 10, 15 }
+Parakeet Pro accepted inputs: { 7, 10, 15 }
+Parakeet ProE accepted inputs: { 10, 15 }
```

Comment:
Maybe we indicate the default values for the settings here in the documentation?

Reply:
Hmm. I have some default value sections in this file, however, those are for optional parameters. I could make some of these settings optional and include a default value. When we first started working on the Parakeet Pro, I know we wanted to user to input their values, and never have default options. Thoughts?

Reply:
Ok.. yeah. When I was thinking defaults I was thinking the values in the launch file we provide. I agree we still don't want built-in defaults where the user can just leave out a parameter and get it to still work. I'm not convinced there's anything to do here. Looking at the Building and Running doc, we say:

> this launch file will launch the ROS node with ***the*** default runtime parameters
(emphasis added)

Are those values in the launch file default values? Idk that they are, since they must be specified (I can't not have them). Maybe they're more like suggested values?